lib/core-classes/DecodersService.ts

```diff
@@ -1,31 +1,64 @@
-import { PluginContext, EmbeddedSDK, Plugin } from "kuzzle";
+import { PayloadService } from './PayloadService';
```

Comment:
Usually we sort the import like this:

1. nodejs import ( `fs`)
2. external lib (`kuzzle`)
3. internal lib (`./lib/...`)